README.md

```diff
@@ -79,6 +79,19 @@ To build:
 gradle dist
 ````
 
+To clone and build all dependencies:
```

Comment:
put this step before build (gradle dist) since it should be executed before building CFI